app/controllers/courses/accredited_body_controller.rb

```diff
@@ -33,7 +33,7 @@ def search_new
       build_provider
       build_previous_course_creation_params
       @query = params[:query]
-      @provider_suggestions = ProviderSuggestion.suggest(@query)
```

Comment:
`ProviderSuggestion.suggest`

that can be blasted from code base?

Reply:
this it might be the case. so i'll make `suggest` do what `suggest_any` is doing and get rid of `suggest_any`

Reply:
i feel that would be another PR though